Sources/MapboxMaps/Foundation/Cancelable.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+/// A type that conforms to `Cancelable` typically represents a long
+/// running operation that can be canceled.
+public protocol Cancelable {
```

Comment:
Should be an `AnyObject` protocol. I don't think it'd make much sense for a struct to conform to this.